src/main/java/core/basesyntax/service/FruitServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dto.OperationType;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class FruitServiceImpl implements FruitService {
+    @Override
+    public void addTransaction(Map<OperationType, OperationHandler> handlerMap,
+                               List<Transaction> transactionList) {
```

Comment:
let's pass `handlerMap` in the constructor instead and make it a field

Reply:
changed